### ISSUE TITLE
README omission

### ISSUE THREAD

**Issue by JacksonBates** [Pairboard/Pairboard#24]
The README should include the prerequisites for initial install (i.e. Node, Mongo).

These are included in the CONTRIBUTING.md, but it would make sense to have them in the README too.

**Comment by BenGitter** [comment]
CONTRIBUTING.md should also include info on how to setup GitHub authentication. Or we have to find a nice way to disable authentication by setting an environment to false 😃

**Comment by JacksonBates** [comment]
Yeah - I haven't had time to think about the auth stuff and how to distribute that for development purposes.

Presumably we can mock auth / non-auth users under dev environment conditions without touching github at all. Github auth only really needs to happen when it's deployed.

I'll give it some more thought, but happy to hear more ideas.

**Comment by JacksonBates** [comment]
Ok, I'm going to add information to the CONTRIBUTING and README.md - the short version is that I think new contributors should set up their own github app for auth in order to generate their own keys. This way we don't have live api keys floating around a whole bunch of one-off contributors and we can keep the proper Pairboard branded github auth app for the eventually deployed app.

**Comment by JacksonBates** [comment]
This is done @BenGitter - see if you can follow the instructions and get it working on your system when you get a chance.

Leaving the original issue open in case we need to get a new contributor on board and they need an easy first PR.

**Comment by BenGitter** [comment]
I directly fixed it when I saw you added authentication, but the instructions should work fine. 

One little thing: 
```
10. Add the line .env to your .gitignore file
```
`.gitignore` is in the repo and the same for everyone else, right?
